akka-stream/src/main/scala/akka/stream/impl/fusing/Ops.scala

```diff
@@ -854,15 +854,21 @@ private[stream] class GroupedWithin[T](n: Int, d: FiniteDuration) extends GraphS
   }
 }
 
-private[stream] class Delay[T](d: FiniteDuration, strategy: DelayOverflowStrategy) extends SimpleLinearGraphStage[T] {
-
+private[stream] final class Delay[T](d: FiniteDuration, strategy: DelayOverflowStrategy) extends SimpleLinearGraphStage[T] {
+  override def initialAttributes: Attributes = DefaultAttributes.delay
+  private[this] val timerName = "DelayedTimer"
```

Comment:
Technically, making it a def *should* put the string literal in the constant pool for the classfile.
I'll change it to a def.